src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
```

Comment:
add access modification: https://www.javatpoint.com/access-modifiers.
Best practice to make fields private.